docs/react-v9/contributing/command-cheat-sheet.md

```diff
@@ -1,4 +1,22 @@
-// TODO #30642 A doc showing a table of every day commands.
+This is a helpful list of helpful every day commands.
 
-Assignee - Mason
-Reviewer - Esteban
+yarn // cleans and installs everything, it fixes many things.
+yarn build // does everything yarn does and updates the API doc file
+yarn buildto package-name // Does a yarn build scoped to a specific package for faster speeds.
+yarn test // runs a test suite. It is slightly different from the test suite in the CI/CD test suite. Can be run from inside a given package to run faster.
+yarn create-package // scaffolds a new package
+yarn create-component // scaffolds a new component
```

Comment:
is this one right? it's barfing in my environment at the moment.

Reply:
It works for me, you have to do it from the root and specify the package. It doesn't work if you try to run it on a specific package.

Reply:
I think you should generally do all the commands from the repo root.